sdk/core/azure-core/inc/azure/core/http/body_stream.hpp

```diff
@@ -170,20 +170,23 @@ namespace Azure { namespace Core { namespace Http {
     }
   };
 
-#ifdef AZ_PLATFORM_POSIX
   /**
    * @brief #BodyStream providing its data from a file.
    */
   class FileBodyStream : public BodyStream {
   private:
-    // in mutable
+#if defined(AZ_PLATFORM_POSIX)
```

Comment:
I am guessing the intent is to say that `m_fd`, `m_baseOffset`, and `m_length` are immutable members, and only m_offset should be mutated. Hence, keep the comment, especially if it applies to `m_hFile` as well.
```suggestion
    // immutable
#if defined(AZ_PLATFORM_POSIX)
```

Reply:
Done.